### DISCUSSION TITLE
APT Package

### DISCUSSION THREAD

**Discussion by aldmbmtl** [AcademySoftwareFoundation/OpenRV#71] [Ideas]
Are there plans to release this for debian/ubuntu? I have a studio running on Ubuntu and it would be great to get that supported if possible?

Thanks!

**Comment by rogernelson** [comment]
We have only tested on CentOS currently, but it is theoretically possible to build on other Linux distros as long as all the build requirements are satisfied.  If you give it a shot, I'd be curious to see your results.

**Comment by aldmbmtl** [comment]
I'll spend some time on it and see where I get 😀
